index.js

```diff
@@ -21,6 +21,7 @@ import Stack from "./src/components/stack";
 import Video from "./src/components/video";
 import formatCredits from "./src/utils/format-credits";
 import formatPowaVideoEmbed from "./src/utils/format-powa-video-embed";
+import getImageFromANS from "./src/utils/get-image-from-ans/getImageFromANS";
```

Comment:
Can we rename the file `getImageFromANS.js` to `index.js` and then we can do an image like the others 

```
import getImageFromANS from "./src/utils/get-image-from-ans";
```

Reply:
/agree

Reply:
@matthewroach , @nschubach Done!